sdk/pinocchio/src/sysvars/clock.rs

```diff
@@ -1,5 +1,7 @@
 //! Information about the network's clock, ticks, slots, etc.
 
+#![allow(deprecated)]
```

Comment:
Why is this needed? For the `get_clock` syscall? If so, can you specify to remove the annotation once the generic `get_sysvar` is used?